### PR TITLE
Update docblocks with official documentation

### PR DESCRIPTION
**Pull request by shadowhand**
Interface documentation was not updated after PSR-15 was approved, which makes these interfaces out of sync with the accepted specification.

Refs php-fig/fig-standards#1107

### CODE REVIEW

**Comment by shadowhand** [comment]
If approved, this should be tagged as version `1.0.1`.

**Review by Jean85 — APPROVED** [review]
Here too, LGTM :+1: in reference to https://github.com/php-fig/fig-standards/pull/1107#issuecomment-434039086

**Comment by Jean85** [comment]
Ping me if you do not have permissions for merge/release.

**Comment by shadowhand** [comment]
I'm not sure who has ownership over these repos. Maybe @weierophinney ?

**Comment by Jean85** [comment]
As editor yes, he has access.

**Comment by weierophinney** [comment]
Um, @shadowhand is editor; I'm sponsor. :smile:

I don't have write access to this repo, based on what GitHub is reporting to me currently...

**Comment by Jean85** [comment]
Woops sorry, I mixed those up.

Well, the editor should retain access to the repos, even after approval; hence, I've given write access to @shadowhand

**Pull request merged**
